4.analyze-components/scripts/util.py

```diff
@@ -16,20 +16,30 @@
 import scripts.cca_core as cca
 
 
-def read_in_z(dataset, z_dim, algorithm, shuffled_data=False):
+def read_in_matrices(dataset, z_dim, algorithm, shuffled_data=False,
+                     load_weights=False):
     """
-    Read in dataset, algorithm, and dimensionality specific z matrix
+    Read in dataset, algorithm, and dimensionality specific z or weight matrix
 
     Arguments:
     dataset - a string indicating either "TCGA", "TARGET", or "GTEX"
     z_dim - a string indicating the bottleneck dimensionality
     algorithm - a string indicating which algorithm to focus on (can be 'all')
     shuffled_data - a boolean if the data was first shuffled before training
+    read_weights - a boolean if the weight matrices should be loaded instead
```

Comment:
instead of what?  what's the default here?  Is this the gene activity that you mentioned?

Reply:
instead of loading in the z matrices. I will clarify in the next commit